README.md

```diff
@@ -42,10 +42,11 @@ Calculate the oscillation period *T = 2π sqrt(L/g)* of a pendulum with length *
 "0.89714 s"
 ```
 
-## Installation
+## Development
 ```
-bower install purescript-quantities
-npm install decimal.js
+npm install
+spago build
```

Comment:
I'm not sure, but I believe that `spago test` already builds modules. Could you please check?

Reply:
Right. removing the `spago build` line...